{{cookiecutter.app_name}}/README.md

```diff
@@ -12,7 +12,10 @@
 
 ### Development
 
-*Development instruction goes here ...*
+- Start development server
```

Comment:
@gutakk Could we add an empty space? We usually add an empty line before code snippet 
```suggestion
- Start development server

```

Reply:
Absolutely. Fixed in c264c5147bfd5692f0233518ba2c18ffc84c0c43